README.md

```diff
@@ -14,4 +42,104 @@ export const CompareText = () => {
   )
 }
 ```
+### To use `useTextComparison` hook
+```js
+import { useTextComparison } from 'text-compare';
+
+const TextDiffer = () => {
+  const text1 = 'The quick brown fox jumps over the lazy dog';
+  const text2 = 'The quick blue fox leaps over the happy dog';
+
+  const { comparisonResult, similarity } = useTextComparison(text1, text2);
+
+  return (
+    <>
+      <div className="flex gap-1 mb-4">{comparisonResult}</div>
+      <div>Similarity: {similarity.toFixed(2)}%</div>
+    </>
+  );
+};
+```
+
+## Advanced Usage 🔧
+
+### Customizing Colors
+
+The `useTextComparison` hook accepts custom colors for highlighting differences:
+
+```jsx
+const customColors = {
+  commonColor: '#1E90FF',    // DodgerBlue for common words
+  removedColor: '#FF6347',   // TomatoRed for removed words
+  addedColor: '#32CD32',     // LimeGreen for added words
+};
+
```

Comment:
Create this as an options objects and pass the cutomColors into it, below pass the options itself.. Doing so, we will be able to add more options to the readme without making considerable changes.

Reply:
Added as per the feedback above.